src/main/java/fi/aalto/cs/apluscourses/intellij/actions/ShowFeedbackAction.java

```diff
@@ -0,0 +1,194 @@
+          if (style != null) {
+            style.remove();
+          }
+          document.head().append(
+              "<style>body {  color: " + textColorString
+                  + ";  font-family: \"" + fontName
+                  + "\", sans-serif; font-weight: 500; padding: 30px; }"
+                  + ".glyphicon:before {  font-weight: bold;  font-size: 20; }"
+                  + ".glyphicon-minus:before {  content: \"-\"; }"
+                  + ".glyphicon-plus:before {  content: \"+\"; }"
+                  + ".label {  display: inline;  padding: .2em .6em .3em;  font-size: 75%;  font-weight: 700;"
+                  + "line-height: 1;  color: #fff;  text-align: center;  white-space: nowrap;"
+                  + "vertical-align: baseline;  border-radius: .25em; }"
+                  + ".label-success {  background-color: #00803c; }"
+                  + ".label-danger {  background-color: #a50000; }"
+                  + "pre {  display: block;  padding: 10.5px;  margin: 0 0 11px;  font-size: 15px;"
+                  + "line-height: 1.428571429;  color: #333;  word-break: break-all;  word-wrap: break-word;"
+                  + "background-color: #f5f5f5;  border: 1px solid #ccc;  border-radius: 4px;  width: fit-content; }"
+                  + "code { padding: 2px 4px; font-size: 90%; color: #c7254e; background-color: #f9f2f4;"
+                  + "border-radius: 4px }"
+                  + ".alert {  padding: 15px;  margin-bottom: 22px;  border: 1px solid transparent;"
+                  + "border-top-color: transparent;  border-right-color: transparent;"
+                  + "border-bottom-color: transparent;  border-left-color: transparent;  border-radius: 4px;"
+                  + " width: fit-content; }"
+                  + ".alert-warning {  color: #8a6d3b;  background-color: #fcf8e3;  border-color: #faebcc; }"
+                  + "ul.last-red>li:last-child {  width: fit-content;  padding: 2;  border-radius: .25em; "
+                  + "background-color: #a50000;  color: white; }</style>");
+          break;
+        case "Programming Studio 2/A":
+          document.head().append("<style>body {  color: " + textColorString
+              + ";  font-family: \"" + fontName + "\", sans-serif; font-weight: 500; padding: 30px; }"
+              + "pre { display: block; padding: 10.5px; margin: 0 0 11px; font-size: 15px; line-height: 1.428571429;"
+              + "color: #333; word-break: break-all; word-wrap: break-word; background-color: #f5f5f5;"
+              + "border: 1px solid #ccc; border-radius: 4px;}</style>");
+          break;
+        default:
+          throw new IllegalStateException("Unexpected value: " + course.getName());
+      }
+
+      var fileEditorManager = FileEditorManager.getInstance(project);
+
+      Arrays.stream(fileEditorManager.getAllEditors())
+          .filter(editor -> editor.getFile() != null && editor.getFile().getName().startsWith("Feedback")).findFirst()
```

Comment:
Is there safer way to recognize feedback editors than `startsWith("Feedback")`. Perhaps even extending HTMLEditor to make FeedbackEditor class and using that one for feedbacks (and making sure there's only one instance open at a time).

Reply:
I changed it to "Feedback for ", which should be specific enough